src/main/scala/backends/futil/FutilBackend.scala

```diff
@@ -339,13 +393,17 @@ private class FutilBackendHelper {
         val struct = List(
           sqrt,
           Connect(argOut.port, sqrt.id.port("in")),
-          Connect(ConstantPort(1, 1), sqrt.id.port("go"))
+          Connect(
+            ConstantPort(1, 1),
+            sqrt.id.port("go"),
+            Some(Not(Atom(sqrt.id.port("done"))))
+          )
         )
         EmitOutput(
           sqrt.id.port("out"),
           sqrt.id.port("done"),
           argOut.structure ++ struct,
-          Some(1)
+          Some(17)
```

Comment:
No magic numbers allowed.

Reply:
this is just the static time of the sqrt component. I'll leave a comment

Reply:
Can you make a `Map` or something and add a key-value pair to it. I assume mult also has a similarly hard coded number somewhere.